website/docs/users-sources/user/user_ref.mdx

```diff
@@ -17,15 +17,20 @@ The User object has the following properties:
 - `path`: User's path, see [Path](#path)
 - `attributes`: Dynamic attributes, see [Attributes](#attributes)
 - `group_attributes()`: Merged attributes of all groups the user is a member of and the user's own attributes. Read-only.
-- `ak_groups`: This is a queryset of all the user's groups.
+- `ak_groups`: This is a queryset of all the user's direct groups.
+- `all_groups()`: This is a queryset of all the user's direct and indirect groups.
+
+:::note Clarifying direct vs indirect group membership
+A user is a member of a group (direct member). That group in turn has a parent group. The user is an indrect member of that parent group.
```

Comment:
Does a user always have an indirect group? For example, if I am in the authentik Admins group, and no other group...

Reply:
there is no indirect group if the direct group has no parent